src/mpid/ch4/generic/am/mpidig_am_part_utils.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#ifndef MPIDIG_AM_PART_UTILS_H_INCLUDED
+#define MPIDIG_AM_PART_UTILS_H_INCLUDED
+
+#include "ch4_impl.h"
+
+/* Request status used for partitioned comm.
+ * Updated as atomic cntr because AM callback and local operation may be
+ * concurrently performed by threads. */
+#define MPIDIG_PART_REQ_UNSET 0 /* Initial status set at psend|precv_init */
+#define MPIDIG_PART_REQ_CTS 2   /* Indicating receiver is ready to receive data
```

Comment:
Even it isn't used, I think we should define `1`. It just read weird with the gap

Reply:
Defined `MPIDIG_PART_REQ_PARTIAL_ACTIVATED` and added note to explain the unused intermediate status.